### DISCUSSION TITLE
Role popup - is tenant specific or instance(sunbird saas)?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#451] [Sunbird ED configuration discussion] [answered]
Can we remove the Student and Parent role from here? 
**If yes then please guide us on how we can do that.**
![image](https://user-images.githubusercontent.com/109504832/228734269-a06c1158-ae5f-428c-9582-762640785659.png)
[@vinukumar-vs](https://github.com/vinukumar-vs)  [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by swayangjit** [comment, marked as answer]
@sachinb80 You can manage these roles by updating the following form

```
"request": {
    "type": "config",
    "subType": "userType",
    "action": "get",
    "component": "portal"
  }
```

**Reply by nidhikachhawa24** [reply]
@swayangjit 
Thank you.
We're able to manage the roles now
